src/client/css/nav.css

```
@@ -19,6 +19,7 @@ nav.site-nav {
       #0000 97%
     )
     1;
+  overflow: scroll;
```

Juuust in case we inadvertently get horizontal overflow as well.

```suggestion
  overflow-y: scroll;
```